src/QuDiffHHL.jl

```diff
@@ -0,0 +1,101 @@
+export array_qudiff, prepare_init_state, LDEMSAlgHHL, bval, aval
+
+"""
+    Based on : arxiv.org/abs/1010.2745v2
+
+    * array_qudiff(N_t,N,h,A) - generates matrix for k-step solver
+    * prepare_init_state(b,x,h,N_t) - generates inital states
+    * solve_qudiff - solver
+
+    x' = Ax + b
+
+    * A - input matrix.
+    * b - input vector.
+    * x - inital vector
+    * N - dimension of b (as a power of 2).
+    * h - step size.
+    * tspan - time span.
+"""
+
+"""
+    LDEMSAlgHHL
+    * step - step for multistep method
+    * α - coefficients for xₙ
+    * β - coefficent for xₙ'
+"""
+
+function bval(alg::LDEMSAlgHHL,t,h,g::Function)
```

Comment:
Is there any reason why the function arg is the last arg? if it is the first arg, then we could use `do` block later.

Reply:
No, no reason. I don't think I can use `do` here. `g` is a function we get from the user (It is input vector 'b' as a function of time in time, in this case).

Reply:
Then do block is even more necessary, since it is a syntax sugar for user to input large functions.

Reply:
Uh. I am not sure how this is to be done. Can you guide me a little?

Reply:
Simply make the function as your first argument. Maybe reading the doc of do block would help?

Reply:
I realise how it is to be done. I've made the change. Please let me know if you think it's fine.

Reply:
Yes, that's basically how you do it. Yes I think this is good now.